src/scripts/demo/index.js

```diff
@@ -49,10 +49,10 @@ window.addEventListener( 'load', () => {
 	 *  Adding an event listener to the form with the id of `wp-notification-metabox-form` that handles "clear all notifications"
 	 */
 	const wpNotificationClearAll = document.getElementById(
-		'clear-all-wp-notify'
+		'clear-all-wp-notifications'
```

Comment:
👍 good spot

Reply:
Also `Cmd+Shift+F` is pretty good on VSCode to find all instances of a word... would never have been able to do it alone.